templates/events-my-events.php

```diff
@@ -33,26 +33,24 @@
 			$event_url                     = gp_url( wp_make_link_relative( $permalink ) );
 			$event_edit_url                = gp_url( 'events/edit/' . $event_id );
 			$event_status                  = get_post_status( $event_id );
-			$event_start                   = ( new DateTime( get_post_meta( get_the_ID(), '_event_start', true ) ) )->format( 'M j, Y' );
-			$event_end                     = ( new DateTime( get_post_meta( get_the_ID(), '_event_end', true ) ) )->format( 'M j, Y' );
-			$event_end_utc                 = new DateTime( get_post_meta( get_the_ID(), '_event_end', true ), new DateTimeZone( 'UTC' ) );
-			$now                           = new DateTime( 'now', new DateTimeZone( 'UTC' ) );
+			$event_start                   = new Event_Start_Date( get_post_meta( get_the_ID(), '_event_start', true ) );
+			$event_end                     = new Event_End_Date( get_post_meta( get_the_ID(), '_event_end', true ) );
 			$stats_calculator              = new Stats_Calculator();
 			$has_stats                     = $stats_calculator->event_has_stats( get_post() );
 
 			?>
 			<li class="event-list-item">
 				<a class="event-link-<?php echo esc_attr( $event_status ); ?>" href="<?php echo esc_url( $event_url ); ?>"><?php the_title(); ?></a>
-				<?php if ( $event_end_utc > $now && ! $has_stats ) : ?>
+				<?php if ( $event_end->is_in_the_past() && ! $has_stats ) : ?>
```

Comment:
The event can't be a past event to be able to edit it.
```suggestion
				<?php if ( ! $event_end->is_in_the_past() && ! $has_stats ) : ?>
```

Reply:
Nice thanks! I think I might had disabled it during debug. I think it's easier on the eye than a comparison with `$now`.